Header header
uint32[] data
geometry_msgs/Point32 center
geometry_msgs/Point32 grid_size
geometry_msgs/Point32 resolution
int32 occupancy_threshold

